models/schema.yml

```diff
@@ -47,6 +55,120 @@ models:
       fal:
         scripts:
           - fal_scripts/anomaly_detection.py
+  - name: fal_views_model
+    description: fal-ai/fal Github Views
+    columns:
+      - name: count
+        description: Total views in one day
+        meta:
+          dimension:
+            type: number
+          metrics:
+            daily_views:
+              type: number
+            total_views:
+              type: sum
+      - name: uniques
+        description: Total unique views in one day
+        meta:
+          dimension:
+            type: number
+          metrics:
+            daily_unique_views:
+              type: number
+      - name: timestamp
+        description: Total views in one day
+        meta:
+          dimension:
+            type: timestamp
+    config:
+      materialized: table
+    meta:
+      owner: "@omer"
```

Comment:
I think `owner` is not used, right? Maybe we can use it for Slack reporting.

Reply:
Ok, will add that.